src/components/CalloutBanner.tsx

```diff
@@ -11,7 +11,7 @@ export type CalloutBannerProps = FlexProps & {
   children?: React.ReactNode
   image: ImageProps["src"]
   imageWidth?: ImageProps["width"]
-  maxImageWidth?: number
+  maxImageWidth?: string
```

Comment:
`maxImageWidth` is being applied to a Chakra attribute (`maxW`), and `imageWidth` is being used for a Next Link attribute (`width`, as changed below instead of `w`)...

```
imageWidth?: number
maxImageWidth?: ImageProps["width"]
```
This seems like it would be more appropriate here to make sure the types match where they're being used